### ISSUE TITLE
[DOTA 2 SLI] Huge drop in FPS while using SLI mode

### ISSUE THREAD

**Issue by icebryanchan** [ValveSoftware/Dota-2#933]
Hi,
I am using Lenovo y500 for years now and it has dual GT750M. When dota was using source 1 I could use SLI to achieve 120fps without any sweat but since source 2 was released, I couldnt use SLI at all as my fps drops to 10 or 15 all the time.

I have been using single GPU for some time now with lowered settings but I really want to get the max experience like back then. That's why I bring up such old issue again. 

However I really hope that I can get my hope so I can use SLI in dota2 again. 

Thank you.

**Comment by gdrewb-valve** [comment]
Can you post your system information?

I don't believe Dota 2 has any SLI support.

**Comment by icebryanchan** [comment]
Yes, before reborn patch it did support SLI. 

Computer Information:
    Manufacturer:  LENOVO
    Model:  20193
    Form Factor: Desktop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Family:  0x6
    CPU Model:  0x3a
    CPU Stepping:  0x9
    CPU Type:  0x0
    Speed:  2395 Mhz
    8 logical processors
    4 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Supported

Network Information:
    Network Speed:  

Operating System Version:
    Windows 10 (64 bit)
    NTFS:  Supported
    Crypto Provider Codes:  Supported 311 0x0 0x0 0x0

Video Card:
    Driver:  NVIDIA GeForce GT 750M

```
DirectX Driver Name:  nvd3dum.dll
Driver Version:  10.18.13.6839
DirectX Driver Version:  10.18.13.6839
Driver Date: 2 June 2016
OpenGL Version: 4.5
Desktop Color Depth: 32 bits per pixel
Monitor Refresh Rate: 59 Hz
DirectX Card: NVIDIA GeForce GT 750M
VendorID:  0x10de
DeviceID:  0xfe4
Revision:  0xa1
Number of Monitors:  1
Number of Logical Video Cards:  1
SLI Detected (2 GPUs)
Primary Display Resolution:  1536 x 864
Desktop Resolution: 1536 x 864
Primary Display Size: 13.58" x 7.64"  (15.55" diag)
                                        34.5cm x 19.4cm  (39.5cm diag)
Primary Bus: PCI Express 8x
Primary VRAM: 2047 MB
Supported MSAA Modes:  2x 4x 8x 
```

Sound card:
    Audio device: Speakers (Realtek High Definiti

Memory:
    RAM:  8142 Mb

Miscellaneous:
    UI Language:  English
    Microphone:  Not set
    Media Type:  DVD
    Total Hard Disk Space Available:  1067736 Mb
    Largest Free Hard Disk Block:  399539 Mb
    OS Install Date: Jan 01 1970
    Game Controller: None detected
    VR Headset: None detected

**Comment by gdrewb-valve** [comment]
This forum is for Mac and Linux bugs so you'll want to post at dev.dota2.com
